### ISSUE TITLE
Peer dependencies warning: marked-terminal requires marked@^0.4.0

### ISSUE THREAD

**Issue by thibaudcolas** [semantic-release/semantic-release#905]
## Current behavior

Since the latest release, I get the following warning on `npm install semantic-release@latest`:

```
npm WARN marked-terminal@3.0.0 requires a peer of marked@^0.4.0 but none is installed. You must install peer dependencies yourself.
```

I haven't noticed any other issue caused by this dependency mismatch.

## Expected behavior

I would expect no peer dependencies warning for dependencies of `semantic-release`

## Environment

- **semantic-release** version: ^15.9.9
- CI environment: local machine, macOS 10.13
- Plugins used, configuration, CI logs: irrelevant

------

This was introduced in #901. As far as I can tell, the underlying problem is https://github.com/greenkeeperio/greenkeeper/issues/278.

**Comment by pvdlg** [comment]
This is because you probably have a lockfile that forces an outdated version of `marked-terminal`.

**Comment by thibaudcolas** [comment]
@pvdlg this happens on an install of `semantic-release` without any existing dependencies or lockfiles. For example in a fresh, empty directory:

```
17:27:44 :Dev/playground
$ cd test
./  ../
17:27:46 :playground/test
$ npm install semantic-release@latest
npm WARN saveError ENOENT: no such file or directory, open '[...]/test/package.json'
npm WARN enoent ENOENT: no such file or directory, open '[...]/test/package.json'
npm WARN marked-terminal@3.0.0 requires a peer of marked@^0.4.0 but none is installed. You must install peer dependencies yourself.
npm WARN test No description
npm WARN test No repository field.
npm WARN test No README data
npm WARN test No license field.

+ semantic-release@15.9.9
added 790 packages in 14.554s
```

This is because `marked-terminal@3.0.0`'s peer dependency is set to marked `^0.4.0`, so doesn't cover `0.5.0`.

Since then, I see that `marked-terminal` has made a [v3.1.0](https://github.com/mikaelbr/marked-terminal/releases/tag/v3.1.0) release which fixes this.

**Comment by pvdlg** [comment]
Semantic-release defines its dependency to marked-terminal as ^3.0.0 so if you don’t have a lock file it should install 3.1.0

**Comment by thibaudcolas** [comment]
Yes, this is now fixed as of 2018-08-24T16:39:51.957Z when `marked-terminal` got released to npm. 

People who had installed `semantic-release@15.9.9` between its release and that of `marked-terminal` just need to reinstall `semantic-release` so its dependency is updated.

@pvdlg this isn't a support issue though – semantic-release isn't supposed to update dependencies outside of their peer dependencies' range, right?
